### DISCUSSION TITLE
This project feels like abandonware.

### DISCUSSION THREAD

**Discussion by kevinh-autoserve1** [the-guild-org/apollo-angular#1774] [General]
Since this project moved under The Guild, it has started to feel more and more like abandonware.
It's extremely disappointing. 
Why is the maintainer MIA, and why can't this project be moved to someone who can interact with the community better?
Is there any hope this will get fixed or should we just move on?

**Comment by hexdecimal16** [comment]
I suggest if you can help in anyway then create PR
